client/components/scoresheet/scoresheet_page/scoresheet_page_controller.js

```diff
@@ -1,9 +1,20 @@
 import Promise from 'bluebird'
 
 class ScoresheetPageController {
-  constructor (scoresheet, scores, logger, user, $scope, $location, $timeout, $window, notifications) {
-    Object.assign(this, { data: scoresheet, scores, logger, user, $scope, $location, $timeout, $window, notifications })
+  constructor (scoresheet, scores, logger, user, $scope, $location, $timeout, $window, notifications, localSettings) {
+    Object.assign(this, { data: scoresheet, scores, logger, user, $scope, $location, $timeout, $window, notifications, localSettings })
     this.ready = false
+    this.scrollDisabled = this.isScrollDisabled()
+    this.localSettings.update('scoresheet-autoscroll', { value: !this.scrollDisabled, type: 'boolean' })
+  }
+
+  isScrollDisabled () {
+    const savedAutoscroll = this.localSettings.getFromLocalStorage('scoresheet-autoscroll')
```

Comment:
Now that you have the `get` method in the `localSettings` service, you should use it insted of the `getFromLocalStorage` and remove the vairable here, since it won't be needed anymore.